src/Discord.Net.Commands/CommandServiceConfig.cs

```diff
@@ -16,6 +17,30 @@ public class CommandServiceConfig
         /// <summary> Determines whether RunMode.Sync commands should push exceptions up to the caller. </summary>
         public bool ThrowOnError { get; set; } = true;
 
+        /// <summary> Collection of aliases that can wrap strings for command parsing. 
+        /// represents the opening quotation mark and the value is the corresponding closing mark.</summary>
+        public Dictionary<char, char> QuotationMarkAliasMap { get; set; }
+        = new Dictionary<char, char>()
```

Comment:
Redundant parentheses here. Do we need the type name too?

Reply:
Type name is required until dotnet/csharplang#100 lands. Do we omit the parentheses in other places too?

Reply:
GitHub doesn't seem to show this as outdated correctly, but please see commit 8187f35975031971b17818aae516fdfaf5d2eb36